addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -560,9 +560,8 @@ export class PropertiesField extends Component {
         const dialogProps = {
             title: _t("Delete Property Field"),
             body: _t(
-                'Are you sure you want to delete this property field? It will be removed for everyone using the "%s" %s.',
-                this.parentName,
-                this.parentString
+                'Are you sure you want to delete this property field? It will be removed for everyone using the "%(parentName)s" %(parentString)s.',
+                { parentName: this.parentName, parentString: this.parentString }
```

Comment:
discussed with @std-odoo
```suggestion
                'Are you sure you want to delete this property field? It will be removed for everyone using the "%(parentName)s" %(parentFieldLabel)s.',
                { parentName: this.parentName, parentFieldLabel: this.parentString }
```